pt-BR/mail/chrome/messenger/focusPane.dtd

```diff
@@ -1,73 +1,73 @@
+<!ENTITY focusPaneSearchAttribute.label        "Atributos">
+<!ENTITY focusPaneSearchAttributeMac.label     "ATRIBUTOS">
+<!ENTITY focusPaneHasAttachmentAttribute.label           "Tem anexo">
+<!ENTITY focusPaneUnreadAttribute.label           "Não-lido">
+<!ENTITY focusPaneFilterByDate.label    "Data">
+<!ENTITY focusPaneFilterByDateMac.label "DATA">
+<!ENTITY focusPaneAnyDate.label         "Qualquer data">
+<!ENTITY focusPaneToday.label           "Hoje">
+<!ENTITY focusPaneYesterday.label       "Ontem">
+<!ENTITY focusPaneThisWeek.label        "Esta semana">
+<!ENTITY focusPaneThisMonth.label       "Este mês">
+<!ENTITY focusPaneFilterByContact.label    "Contatos favoritos">
+<!ENTITY focusPaneFilterByContactMac.label "CONTATOS FAVORITOS">
 
 <!-- focus pane topics context menus -->
-<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Rename">
+<!ENTITY pbFavoriteTopicsContextMenu-edit.label         "Renomear">
 <!ENTITY pbFavoriteTopicsContextMenu-edit.accesskey     "n">
-<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remove From Favorites">
+<!ENTITY pbFavoriteTopicsContextMenu-favorite.label     "Remover dos favoritos">
 <!ENTITY pbFavoriteTopicsContextMenu-favorite.accesskey "R">
-<!ENTITY pbFavoriteTopicsContextMenu-delete.label       "Delete">
+<!ENTITY pbFavoriteTopicsContextMenu-delete.label       "Excluir">
 <!ENTITY pbFavoriteTopicsContextMenu-delete.accesskey   "D">
-<!ENTITY pbFavoriteTopicsContextMenu-add.label          "Create New Favorite Topic">
+<!ENTITY pbFavoriteTopicsContextMenu-add.label          "Criar novo tópico favorito">
 <!ENTITY pbFavoriteTopicsContextMenu-add.accesskey      "C">
-<!ENTITY pbFavoriteTopicsContextMenu-editAll.label      "Edit Topics...">
+<!ENTITY pbFavoriteTopicsContextMenu-editAll.label      "Editar tópicos...">
 <!ENTITY pbFavoriteTopicsContextMenu-editAll.accesskey  "t">
 
 <!-- focus pane favorite contacts context menus -->
-<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remove From Favorites">
+<!ENTITY pbFavoriteContactsContextMenu-delete.label     "Remover dos favoritos">
 <!ENTITY pbFavoriteContactsContactsMenu-delete.accesskey "R">
-<!ENTITY pbFavoriteContactsContextMenu-email.label       "Compose">
+<!ENTITY pbFavoriteContactsContextMenu-email.label       "Criar">
 <!ENTITY pbFavoriteContactsContactsMenu-email.accesskey  "e">
-<!ENTITY pbFavoriteContactsContextMenu-edit.label        "Edit Contact...">
+<!ENTITY pbFavoriteContactsContextMenu-edit.label        "Editar contato...">
 <!ENTITY pbFavoriteContactsContactsMenu-edit.accesskey   "d">
-<!ENTITY pbFavoriteContactsContextMenu-createfilter.label "Create Filter...">
+<!ENTITY pbFavoriteContactsContextMenu-createfilter.label "Criar filtro...">
 <!ENTITY pbFavoriteContactsContactsMenu-createfilter.accesskey "C">
 
-<!ENTITY pbFavoriteContacts.DragLabel "Drag your favorite contacts here">
+<!ENTITY pbFavoriteContacts.DragLabel "Arraste seus contatos favoritos aqui">
```

Comment:
I agree with the addition of para, but it's weird with aqui = "para aqui". I'll change to "para cá". Please let me know if you agree.